src/js/core/polyfills.js

```diff
@@ -102,11 +102,42 @@ function domPolyfills() {
     })([Element.prototype, CharacterData.prototype, DocumentType.prototype]);
 }
 
+function JSXPolyfills() {
+    // https://betterprogramming.pub/how-to-use-jsx-without-react-21d23346e5dc
```

Comment:
for legal reasons maybe link to [the CodeSandbox](https://codesandbox.io/s/jsx-in-the-browser-qd2hq) and remark that it's MIT, though it seems you stitched it from the article (which consists of unlicensed gists!!!!!!!!!!)

Reply:
replaced with my own code 😎